src/client/.prettierrc

```diff
@@ -1,5 +1,6 @@
 {
   "semi": false,
   "singleQuote": true,
-  "printWidth": 120
+  "printWidth": 120,
+  "trailingComma": "all"
```

Comment:
Changes like this should really be in their own PR if possible.

Reply:
I agree — definitely an oversight.